google/cloud/spanner_v1/session.py

```diff
@@ -470,6 +470,7 @@ def run_in_transaction(self, func, *args, **kw):
         ) as span:
             while True:
                 if self._transaction is None:
+                    add_span_event(span, "Creating Transaction")
```

Comment:
This can be removed since `self.transaction() ` doesn't make an API call. It is just an inmemory trnasaction object that gets created

Reply:
Got it, thanks!